packages/app-builder-backend/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@reapit/app-builder-backend",
+  "version": "0.0.1",
+  "description": "Common Package for Reapit Foundation",
+  "repository": {
+    "type": "git",
+    "url": "git+git@github.com:reapit/foundations.git"
+  },
+  "license": "MIT",
+  "author": "Reapit Ltd",
+  "main": "./src/index.ts",
+  "scripts": {
+    "start": "ts-node .",
+    "test": "echo '...skipping...'",
+    "build": "tsc",
+    "lint": "eslint --cache --ext=ts,tsx,js src --fix",
+    "release": "cdk deploy --require-approval never",
+    "publish": "yarn npm publish --access public",
+    "conf": "echo '...skipping...'",
+    "synth": "cdk synth",
```

Comment:
What does this do? Just interested :-)

Reply:
it generates the cloudformation template from the cdk code and validates it, useful if you make a cdk code change and want to make sure that it'll deploy without having to actually deploy it